src/transformers/file_utils.py

```diff
@@ -59,6 +59,7 @@
     EntryNotFoundError,
     ExplicitEnum,
     ModelOutput,
+    OptionalDependencyNotAvailable,
```

Comment:
No need to add it here as this file only exists for backward compatibility reasons and this is a new object.